packages/debug-utils/index.js

```diff
@@ -530,13 +530,16 @@ var DebugUtils = {
     }
   },
 
-  formatInstruction: function (traceIndex, traceLength, instruction) {
+  formatCurrentInstruction: function (instruction) {
+    const pc = this.formatPC(instruction.pc);
+    const formattedInstruction = this.formatInstruction(instruction);
+    return (
+      "-> " + truffleColors.mint(formattedInstruction) + pc
```

Comment:
Ah, I see you went with no color on the arrow.

Reply:
I tried different colors such as red, purple etc using truffleColors().  At the end, I chose no color.  Please feel free to suggest another color.